target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -163,6 +163,10 @@ pcs,cr3000)
 qihoo,c301)
 	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	;;
+teltonika,rut230)
+       ucidef_set_led_netdev "lan" "LAN" "rut230:green:lan" "eth0"
+       ucidef_set_led_netdev "wan" "WAN" "rut230:green:wan" "eth1"
```

Comment:
For proper link-state detection, I suspect that one statement (probably lan) needs to be set up with ucidef_set_led_switch here. I'd give this low priority, though.